RoaringBitmap/src/main/java/org/roaringbitmap/BitmapContainer.java

```diff
@@ -423,6 +423,19 @@ public boolean equals(Object o) {
       return Arrays.equals(this.bitmap, srb.bitmap);
     } else if (o instanceof RunContainer) {
       return o.equals(this);
+    } else if (o instanceof ArrayContainer) {
```

Comment:
This cannot happen. An ArrayContainer can never be equal to a BitmapContainer when the data abides by the specification.